src/index.html

```diff
@@ -4,9 +4,381 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link 
+      rel="stylesheet" 
+      href="./styles/main.scss"
+    >
+    
+    <link 
+      rel="preconnect" 
+      href="https://fonts.googleapis.com"
+    >
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+    >
+    <link 
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" 
+      rel="stylesheet"
+      crossorigin="anonymous"
+    >
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <img
+        class="header__icon"
+        src="./images/logo.png"
+        alt="header-logo-image"
+      >
+      <nav class="header__nav">
+        <ul class="header__link-cont">
+          <li class="header__link-cont">
+            <a 
+              class="header__nav-link is-active" 
+              href="#"
+            >
+              apple
+            </a>
+          </li>
+          <li class="header__link-cont">
+            <a 
+              class="header__nav-link" 
+              href="#"
+            >
+              samsung
+            </a>
+          </li>
+          <li class="header__link-cont">
+            <a 
+              class="header__nav-link" 
+              href="#"
+            >
+              smartphones
+            </a>
+          </li>
+          <li class="header__link-cont">
+            <a 
+              class="header__nav-link " 
+              href="#" 
+              data-qa="nav-hover"
+            >
+              laptops & computers
+            </a>
+          </li>
+          <li class="header__link-cont">
+            <a 
+              class="header__nav-link" 
+              href="#"
+            >
+              gadgets
+            </a>
+          </li>
+          <li class="header__link-cont">
+            <a 
+              class="header__nav-link" 
+              href="#"
+            >
+              tablets
+            </a>
+          </li>
+          <li class="header__link-cont"> 
+            <a 
+              class="header__nav-link" 
+              href="#"
+            >
+              photo
+            </a>
+          </li>
+          <li class="header__link-cont">
+            <a 
+              class="header__nav-link" 
+              href="#"
+            >
+              video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <div class="container__holder">
+        <div class="card card--1" data-qa="card">
```

Comment:
Consider using more semantic tags for a card such as ```<article></article>```. It's semantic and fits better.